app/src/main/java/io/gnosis/safe/Tracker.kt

```diff
@@ -12,10 +12,6 @@ class Tracker private constructor(context: Context) {
 
     private val firebaseAnalytics: FirebaseAnalytics = FirebaseAnalytics.getInstance(context)
 
-    fun setCurrentScreenId(activity: Activity, screenId: ScreenId) {
-        firebaseAnalytics.setCurrentScreen(activity, screenId.value, null)
```

Comment:
So we are treating screens as if they were any other event?

Reply:
yes, because screen tracking doesn't work on iOS. We are restricted to 500 events (hopefully this will be enough)